src/cookieConsent.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from "react";
+import { Button, Grid, Snackbar } from "@material-ui/core";
+import { DAY_IN_MS } from "./utils/TimeUtils";
+import Cookies from "js-cookie";
+import { initAnalytics } from "./utils/analytics";
+import PropTypes from "prop-types";
+
+export const COOKIE_ACCEPT_LIFETIME_DAYS = 30;
+export const COOKIE_REJECT_LIFETIME_DAYS = 5;
+
+export const CookieConsent = () => {
+    const [open, setOpen] = useState(false);
+
+    const handle = (accepted) => {
+        if (accepted) {
+            initAnalytics();
+        } else {
+            Cookies.remove("_ga");
+            Cookies.remove("_gat");
+            Cookies.remove("_gid");
```

Comment:
Why do we need an extra package just for this? Just use plain js

https://www.guru99.com/cookies-in-javascript-ultimate-guide.html#:~:text=JavaScript%20Delete%20Cookie,expires%20to%20a%20passed%20date.

Reply:
Removed npm dependency. Do I need to commit the whole Package lock?

Reply:
No, you should untrack package lock from these commits, as no dependency changed